### DISCUSSION TITLE
Missing levels in Galagino Donkey Kong

### DISCUSSION THREAD

**Discussion by kbman22** [harbaum/galagino#47] [Q&A]
Hello guys, 
I built the Galagino and everything works so far. Only in Donkey Kong did I notice that the middle two levels (elevator and conveyor belt) were missing. Is this the case with the Galagino or am I making a mistake here? 
I'm grateful for every idea ;)

**Comment by harbaum** [comment]
These are the original ROMs you are running, so the game will behave like the real device. The Emulation is actually not able to alter the gameplay.

The levels you are missing will show up if you progress further into the game.

You might want to have a look at game play videos on YouTube or the like to verify the behaviour.

Some arcades do have dip switch settings, so this behaviour may have been adjustable. You may want to check if this could actually be influenced on a real DK arcade.

**Comment by RH70-0208** [comment]
@kbman22 

Search in file "dkong_rom1.h"
the following byte sequence:
(That should be line 936)
0x01,0x04,0x01,0x03,0x04,0x01,0x02,0x03,0x04,0x01,0x02,0x01,0x03,0x04,0x01,0x02,0x01,0x03,0x01,0x04

Replace them with: 
0x01,0x02,0x03,0x04,0x01,0x02,0x03,0x04,0x01,0x02,0x03,0x04,0x01,0x02,0x03,0x04,0x01,0x02,0x03,0x04
and you have the japan screenorder.

Save ladders patch:
Search the following byte sequence:
(That should be line 538)
0x05,0x62,0xD6,0x04 and change one byte (0x04 -> 0x10)
0x05,0x62,0xD6,0x10 

Then you have activated the save ladders option from the japan version.
Barrels won't fall down the ladder, if Mario is standing on top of it.
